client/src/App.vue

```diff
@@ -2,17 +2,45 @@
     <VApp>
         <RouterView />
         <Notifications />
+        <Alert
+            :visibility="alert.visible"
```

Comment:
you do not need alert, because there already exists Notifications, use them. If you'd like to change a style, add modification